### DISCUSSION TITLE
Looking for paid service for pgsync support

### DISCUSSION THREAD

**Discussion by jayantwaghulde** [toluaina/pgsync#287] [Q&A] [answered]
Hi All,
Does this community have any premium/paid service to support us quickly? I would appreciate it if you could provide that. Please contact me back at jayant.waghulde@gmail.com

**Comment by toluaina** [comment, marked as answer]
I've sent you a message

**Comment by rjmAmaro** [comment]
Hello, @toluaina . I'm currently using pgsync at my job and we're also looking for some paid support on it. Could you provide it? Please contact me at rjamaro[at]redlightsoft[dot]com, if possible.

Thank you
